tests/components/eheimdigital/test_sensor.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for the sensor module."""
+
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from eheimdigital.types import EheimDeviceType, FilterErrorCode
+import pytest
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .conftest import init_integration
+
+from tests.common import MockConfigEntry, snapshot_platform
+
+
+@pytest.mark.usefixtures("classic_vario_mock")
+async def test_setup_classic_vario(
+    hass: HomeAssistant,
+    eheimdigital_hub_mock: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+) -> None:
+    """Test fan platform setup for filter."""
+    mock_config_entry.add_to_hass(hass)
+
+    with (
+        patch("homeassistant.components.eheimdigital.PLATFORMS", [Platform.SENSOR]),
+        patch(
+            "homeassistant.components.eheimdigital.coordinator.asyncio.Event",
+            new=AsyncMock,
+        ),
```

Comment:
Why do we patch this

Reply:
You mean the Event? Because else it will run in timeout while waiting on the asyncio.Event raised when receiving the first message from the device (which is not raised in tests because of no real communication):

https://github.com/home-assistant/core/blob/8aee79085ab159ffe90c866f1d7e402f31d00c3c/homeassistant/components/eheimdigital/coordinator.py#L83-L88